redash/utils.py

```diff
@@ -95,9 +95,9 @@ def default(self, o):
         if isinstance(o, decimal.Decimal):
             return float(o)
 
-        if isinstance(o, datetime.date):
+        if isinstance(o, datetime.date) or isinstance(o, datetime.datetime):
```

Comment:
From what I understand `datetime.datetime` inherits `datetime.date`, and hence why `isinstance(datetime.datetime.now(), datetime.date) == True`. You see different behavior?